### PR TITLE
Update mx.m3u

### LINKED ISSUE #6672: Add: TUDN (México)
### Channel Name

TUDN 

### Channel ID

TUDNMexico.mx

### Website

_No response_

### Stream URL

http://190.61.102.67:2000/play/a08j/index.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #13149: Add: Canal de Las Estrellas | [MX]
### Channel ID

LasEstrellas.mx

### Stream URL (optional)

https://channel01-onlymex.akamaized.net/hls/live/2022749/event01/index.m3u8

### Notes (optional)

Stream works fine with a México VPN IP
Screenshot_20230610_132945_VLC
So the Vix.com linear link might be International 
Linear Version:
Screenshot_20230610_133636_VLC
https://linear-416.frequency.stream/416/hls/master/playlist.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #13151: Add:  Canal 22 | [MX]
### Channel ID

Canal22Nacional.mx

### Stream URL (optional)

https://5fc584f3f19c9.streamlock.net/canal22/smil:canal22.smil/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9My8yNC8yMDIzIDg6MTI6MTQgUE0maGFzaF92YWx1ZT1QTUJuUHAraFpxTVdJNGViWFUyVjVnPT0mdmFsaWRtaW51dGVzPTUmaWQ9

### Notes (optional)

Stream works fine with México VPN IP
Screenshot_20230610_141121_Chrome

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by freearhey**
All three links to Las Estrellas show different streams and I was unable to determine the exact ID for each version: https://iptv-org.github.io/?q=Las%20Estrellas%20country:MX

Closes #6672, closes #13149, closes #13151

### CODE REVIEW

File: streams/mx.m3u [hunk 1]
```diff
@@ -237,3 +237,9 @@ https://5ca9af4645e15.streamlock.net/zacatecas/videozacatecas/playlist.m3u8
 https://channel07-notusa.akamaized.net/hls/live/2036971-b/event01/index.m3u8
 #EXTINF:-1 tvg-id="",TELE SISTEMA CANAL 9 (486p) [Geo-blocked]
 http://k4.usastreams.com/ARBtv/ARBtv/playlist.m3u8
+#EXTINF:-1 tvg-id="TUDNMexico.mx",TUDN Mexico (540p)
+http://190.61.102.67:2000/play/a08j/index.m3u8
+#EXTINF:-1 tvg-id="LasEstrellas.mx",Las Estrellas (720p)
+https://channel01.akamaized.net/hls/live/2022749/event01/index.m3u8
+#EXTINF:-1 tvg-id="Canal22Nacional.mx",Canal22Nacional.mx (720p) [Geo-blocked]
+https://5fc584f3f19c9.streamlock.net/canal22/smil:canal22.smil/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9My8yNC8yMDIzIDg6MTI6MTQgUE0maGFzaF92YWx1ZT1QTUJuUHAraFpxTVdJNGViWFUyVjVnPT0mdmFsaWRtaW51dGVzPTUmaWQ9
```
Comment (BellezaEmporium) [inline streams/mx.m3u:245]:
```suggestion
https://mc_qs.lanesh4d0w.tech/iptv-query?streaming-ip=https://conceptoweb-studio.com/radio/video/canal22/
```

arrgh... here you go.

Comment (BellezaEmporium) [reply streams/mx.m3u:245]: needs a bit of setupping but it'll come soon

File: streams/mx.m3u [hunk 2]
```diff
@@ -237,3 +235,13 @@ https://5ca9af4645e15.streamlock.net/zacatecas/videozacatecas/playlist.m3u8
 https://channel07-notusa.akamaized.net/hls/live/2036971-b/event01/index.m3u8
 #EXTINF:-1 tvg-id="",TELE SISTEMA CANAL 9 (486p) [Geo-blocked]
 http://k4.usastreams.com/ARBtv/ARBtv/playlist.m3u8
+#EXTINF:-1 tvg-id="TUDNMexico.mx",TUDN Mexico (540p)
+http://190.61.102.67:2000/play/a08j/index.m3u8
+#EXTINF:-1 tvg-id="Canal22Nacional.mx",Canal22Nacional.mx (720p) [Geo-blocked]
+http://mc-qs.lanesh4d0w.tech/iptv-query?streaming-ip=https://conceptoweb-studio.com/radio/video/canal22/
+#EXTINF:-1 tvg-id="",Las Estrellas (720p)
+https://channel01.akamaized.net/hls/live/2022749/event01/index.m3u8
```
Comment (BellezaEmporium) [inline streams/mx.m3u:243]: Seems to be the National stream (outside of the US citizens, could be having a timezone change)

File: streams/mx.m3u [hunk 3]
```diff
@@ -237,3 +235,13 @@ https://5ca9af4645e15.streamlock.net/zacatecas/videozacatecas/playlist.m3u8
...
+#EXTINF:-1 tvg-id="",Las Estrellas (480p)
+http://190.61.102.67:2000/play/a00g/index.m3u8
```
Comment (BellezaEmporium) [inline streams/mx.m3u:245]: More local stream, most certainly MX

File: streams/mx.m3u [hunk 4]
```diff
@@ -237,3 +235,13 @@ https://5ca9af4645e15.streamlock.net/zacatecas/videozacatecas/playlist.m3u8
...
+#EXTINF:-1 tvg-id="",Las Estrellas (1080p)
+https://linear-416.frequency.stream/416/hls/master/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/mx.m3u:247]: Seems to be the national stream, but possibly with no timezone program change ?

File: streams/mx.m3u (hunk 2 above)
Comment (freearhey) [reply streams/mx.m3u:243]: Could you provide a specific ID so I can add it to the playlist?

Comment (BellezaEmporium) [reply streams/mx.m3u:243]: Could be LasEstrellas.mx

File: streams/mx.m3u (hunk 3 above)
Comment (BellezaEmporium) [reply streams/mx.m3u:245]: Seems to be XHXTDT1.mx

File: streams/mx.m3u (hunk 4 above)
Comment (BellezaEmporium) [reply streams/mx.m3u:247]: Could also be LasEstrellasLatinAmerica.mx with a +1 or a +2

**Comment by freearhey** [comment]
Updated.

**Review by BellezaEmporium — APPROVED** [review]

**Review by Carlinhos027 — APPROVED** [review]

**Pull request merged**